src/types/multiModel.js

```diff
@@ -83,67 +73,62 @@ class MultiModel extends BaseType {
         throw new Error('MultiModel is virtual. It must be extended, and requestOne() must be overriden');
     }
 
-    requestMany (options: ?Object): void {
+    requestMany (options: Object): void {
         throw new Error('MultiModel is virtual. It must be extended, and requestMany() must be overriden');
     }
 
-    apply (response: any, options: ?Object): any {
-        let models = Array.isArray(response) ? response : [response];
-
-        const idKey = this.Model.idField;
-
-        const ids = models.map((modelData) => {
-            return modelData[idKey];
-        });
-
-        this.requestCache.set(options, ids);
-        this.store(models);
-    }
-
-    get (id: string, options: ?Object): any {
-        let model = this.fetch(id);
+    get (id: string, options: Object = {}): any {
+        let model = this._getModelById(id);
 
         if  (model) {
             return model;
 
         } else {
-            this.requestOne(id, options || {});
+            this.requestOne(id, options);
         }
     }
 
-    all (options: ?Object): Array<any> {
-        let ids = this.requestCache.get(options || {});
+    add (model: Object, index: ?number): Object {
 
-        if (ids) {
-            return ids.map((id) => {
-                return this.fetch(id);
-            });
-        } else {
-            this.requestMany(options || {});
-            return [];
+        if (!this.map) {
+            throw new Error('An unmapped OwnsMany cannot be added to');
```

Comment:
An OwnsMany cant really have items added or removed at all. You can just either create or destroy models in the owns many, but you cant just unlink a performer from an exposition without deleting it.